data/migrations/deb/helper_1_0_96_to_1_0_97.py

```diff
@@ -15,9 +15,9 @@
 from storage.chunked_file_store import ChunkedFileStore
 from stp_core.common.log import getlogger
 
-from sovrin_common.config_util import getConfig
```

Comment:
There should be no changes in migration scripts since they are supposed to be used with an old version of code.

Reply:
Migration scripts should be operable on the target code base.